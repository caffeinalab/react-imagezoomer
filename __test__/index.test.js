const React = require('react');
const renderer = require('react-test-renderer');

const ImageZoomer = require('../src/index');

it('renders correctly', () => {
  const tree = renderer
    .create(<ImageZoomer image="https://source.unsplash.com/random" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});