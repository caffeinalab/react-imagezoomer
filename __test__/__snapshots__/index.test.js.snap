// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <img
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseMove={null}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    src="https://source.unsplash.com/random"
    style={
      Object {
        "backgroundPosition": "center center",
        "backgroundRepeat": "no-repeat",
        "backgroundSize": "contain",
        "height": "auto",
        "position": "relative",
        "width": "100%",
      }
    }
  />,
  <div
    style={
      Object {
        "background": "white",
        "border": "1px solid",
        "borderRadius": "50%",
        "height": "150px",
        "left": "-75px",
        "overflow": "hidden",
        "pointerEvents": "none",
        "position": "absolute",
        "top": "-75px",
        "transform": "translateY(0px) translateX(0px)",
        "visibility": "hidden",
        "width": "150px",
      }
    }
  >
    <div
      style={
        Object {
          "backgroundImage": "url(https://source.unsplash.com/random)",
          "backgroundPosition": "center center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "contain",
          "height": 0,
          "left": "50%",
          "pointerEvents": "none",
          "position": "absolute",
          "top": "50%",
          "transform": "translateY(-0px) translateX(-0px)",
          "transformOrigin": " top left",
          "width": 0,
        }
      }
    />
  </div>,
]
`;
